components/mod-event/useModEventList.tsx

```diff
@@ -321,3 +354,39 @@ export const useModEventList = (
     addToWorkspace,
   }
 }
+
+const TagBasedTypeFilters = {
+  [MOD_EVENTS.DISABLE_DMS]: { add: DM_DISABLE_TAG, remove: '' },
+  [MOD_EVENTS.ENABLE_DMS]: { remove: DM_DISABLE_TAG, add: '' },
+  [MOD_EVENTS.DISABLE_VIDEO_UPLOAD]: {
+    add: VIDEO_UPLOAD_DISABLE_TAG,
+    remove: '',
+  },
+  [MOD_EVENTS.ENABLE_VIDEO_UPLOAD]: {
+    remove: VIDEO_UPLOAD_DISABLE_TAG,
+    add: '',
+  },
+}
```

Comment:
Using objects as a map is not ideal in javascript (because `TagBasedTypeFilters['toString']` will be there and might cause abuse). You can make it safer by using a `Map` (best) or by removing the inheritance of `Object.prototype` like so (technically safe but funkier in terms of maintainability):

```suggestion
const TagBasedTypeFilters = {
  __proto__: null, // Allow using this as a "map"
  [MOD_EVENTS.DISABLE_DMS]: { add: DM_DISABLE_TAG, remove: '' },
  [MOD_EVENTS.ENABLE_DMS]: { remove: DM_DISABLE_TAG, add: '' },
  [MOD_EVENTS.DISABLE_VIDEO_UPLOAD]: {
    add: VIDEO_UPLOAD_DISABLE_TAG,
    remove: '',
  },
  [MOD_EVENTS.ENABLE_VIDEO_UPLOAD]: {
    remove: VIDEO_UPLOAD_DISABLE_TAG,
    add: '',
  },
}
```

Reply:
hmm... agreed maps are better but not convinced we should be worried about this type of "abuse" in this context.